src/install_pypi.rs

```diff
@@ -440,7 +440,7 @@ fn whats_the_plan<'a>(
     let installed = site_packages.iter().filter(|dist| {
         dist.installer()
             .unwrap_or_default()
-            .is_some_and(|installer| installer == "uv")
+            .is_some_and(|installer| installer == "uv-pixi")
```

Comment:
Would be good to turn this into a `const`

Reply:
This: https://github.com/prefix-dev/pixi/pull/1043 already had that and some logic